### DISCUSSION TITLE
AIO Nextcloud installation leaves me with a tiny data container

### DISCUSSION THREAD

**Discussion by diagonalljo** [nextcloud/all-in-one#5244] [Questions] [answered]
I installed AIO nextcloud on a VPS with Oracle Linux and Docker.  The installation was straightforward and I'm impressed with the speed and overall performance.  But the installation reduced my 640 GB VPS to a 69 GB space for my data.  I need more or less 400 GB so I'm stuck with this.  During the installation there were no options for configuring the Nextcloud containers.  How can I expand my data directory or data container?  Do I need to shrink other containers?

**Comment by szaimen** [comment, marked as answer]
Hi, it is weird that AIO needs so much storage space. You can find out what eats your storage space e.g. with the program ncdu on your server.

**Reply by diagonalljo** [reply]
Thanks for the tip.  I'll do that.
In the meantime, here's the result after executing df -H  :
```
df -H
Filesystem                                Size  Used Avail Use% Mounted on
devtmpfs                                  4.2M     0  4.2M   0% /dev
tmpfs                                       8.2G     0  8.2G   0% /dev/shm
tmpfs                                       3.3G  9.7M  3.3G   1% /run
/dev/mapper/ol_v2202305200827229967-root   76G   25G   51G  33% /
/dev/vda1                                 1.1G  474M  534M  48% /boot
/dev/mapper/ol_v2202305200827229967-home  611G  4.3G  607G   1% /home
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/b01a0616da5bb5b4382c6c4294d444dfafe42eb7c4a6f8743f4dd76d423d8f28/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/e10da3d0c5adcb0391c85016d18ba8061f5c391885bd4f449a5d52217e7fdc35/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/35f19fd992f04e3886ed9fb87c818e6ab9cf3fe14bb42cbf9bc492c11e2d9ed4/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/70dceeb88225023690922425640b6598d75d9ad7e372d6c0b0dd7255fe7915f1/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/5df6334442b3c6c9e0dfe4fcafc1f3c273f970f02d9af5e2f00189b8f0952659/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/1be8ce5b86d25fd3212c1043762223763ca4fcdc5064971a69bfb77097f62297/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/d1444e1f26ef07655d88cd467a486844b558c2da845c5ce37bcf1d6b8acb196e/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/27323912bbd2b897bfe971f76fafb06dfedf0281eef76bc87c5d9116edb7dbbe/merged
overlay                                    76G   25G   51G  33% /var/lib/docker/overlay2/b4c77677db446029f6f17f16b82ad3dd90c81e4e2e73cd3887bf45abb501e0a2/merged
tmpfs                                     1.7G     0  1.7G   0% /run/user/1000
```

**Reply by szaimen** [reply]
> Filesystem                                Size  Used Avail Use% Mounted on
> /dev/mapper/ol_v2202305200827229967-root   76G   25G   51G  33% /
> /dev/mapper/ol_v2202305200827229967-home  611G  4.3G  607G   1% /home

Looks like 25G on your root drive and 4.3G on the home drive are used? So not sure what you are looking for?

**Reply by diagonalljo** [reply]
![nextcloud system](https://github.com/user-attachments/assets/123ec1fd-b508-4088-86e0-51e670bf03b6)

It's the "administrations settings" / "system" page on the nextcloud server that worries me.  It says there's only 47 GB space left.

**Comment by diagonalljo** [comment]
I could try to fill my data space beyond this 47 GB limit and see what happens.

**Reply by szaimen** [reply]
Have you set the variable `NEXTCLOUD_DATADIR`? If not, you might actually fill the root disk. See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-default-location-of-nextclouds-datadir

**Reply by diagonalljo** [reply]
Thanks, Szaimen.  I have no time right now to check it out.  I'll give it a try this evening and I'll get back to you.
Jozef.

**Comment by diagonalljo** [comment]
Your suggestion gave me the solution.  I reinstalled nextcloud AIO and set the NEXTCLOUD-DATADIR to /var/lib/nextclouddata. After reinstalling, the disk size became almost 600 GB. Thanks a lot!
